server/graphql/v1/mutations/collectives.js

```diff
@@ -196,6 +196,7 @@ export async function createCollectiveFromGithub(_, args, req) {
     const host = await models.Collective.findByPk(defaultHostCollective('opensource').CollectiveId);
     const promises = [
       collective.addUserWithRole(user, roles.ADMIN),
+      host.addUserWithRole(user, roles.ADMIN),
```

Comment:
:red_circle: this is not safe and can't be merged. Set up the E2E test to bypass the limitation.

Reply:
@kewitz : Maybe I am missing something here, but my thinking is that the preceding `if` block; `if (config.env !== 'production' && user.email.match(/.*test.*@opencollective.com$/))` makes sure this only executes for E2E tests. 

The problem I am trying to solve here is that our E2E tests doesn't support logging in as a host admin (only just a collective admin). Maybe thinking further I think we could add a condition such that when a user passes a test email of the form `testuser+host@opencollective.com` where we can check for the `host` keyword within the test email here and only execute this condition for that particular case. Something like, 

```
if (user.email.match(<host-keyword-match-regex>)){
  host.addUserWithRole(user, roles.ADMIN),
}
```
WDYT? 🤔

Reply:
As discussed I've added a condition to support emails such as `testuser+host@opencollective.com` here. We probably need to update the [documentation](https://docs.opencollective.com/help/contributing/development/testing-with-cypress) as well, I'll do that in a separate PR. 😄